src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let correctActiveTabId = activeTabId;
```

Comment:
better use state for this goals

Reply:
I got too many re-renderings (

Reply:
Additional re-render itself is not a big deal in this case. But if you want to avoid it, it'd be better to make the code more compact. For example:
```
const getCorrectedTabId = (allTabs, tabId) => {
  return allTabs.some(({id}) => id === tabId) ? tabId : allTabs[0].id;
}
...
const correctActiveTabId = getCorrectedTabId(activeTabId);
```